### DISCUSSION TITLE
Rpi5 and GPIO LED

### DISCUSSION THREAD

**Discussion by TomBonk** [BlitterStudio/amiberry#1531] [Q&A] [answered]
Hi
Does the release on Rpi5 have GPIO LED support enabled by default or do I have to compile from source with the option “USE_GPIOD=1”

I'm asking because I'm trying to get these LEDs working, but without success.

libgpiod-dev has been installed.
Amiberry is started automatically from the root account.
The LEDs still do not show any activity.

The LEDs are connected correctly because I turned them on manually using gpioset.

What could be the problem in this case ?

**Comment by midwan** [comment, marked as answer]
It doesn't have that feature enabled currently, no.
You would have to compile it with that option enabled, to get it working. At least for now.
